cognite/client/data_classes/spaces.py

```diff
@@ -0,0 +1,51 @@
+from __future__ import annotations
+
+from typing import TYPE_CHECKING, cast
+
+from cognite.client.data_classes._base import (
+    CogniteResource,
+    CogniteResourceList,
+)
+
+if TYPE_CHECKING:
+    from cognite.client import CogniteClient
+
+_RESERVED_SPACE_IDS = {"space", "cdf", "dms", "pg3", "shared", "system", "node", "edge"}
+
```

Comment:
Could use `frozenset`

Reply:
as it is private to the module, I do not think it is worth the extra import.

Reply:
Import? It’s built in 😉